src/services/problems.ts

```diff
@@ -0,0 +1,16 @@
+import CodeforcesClient from '@acmascis/codeforces-client';
+import { Problem } from '@acmascis/codeforces-client/build/interfaces/problem.interface';
+import { Standings } from '@acmascis/codeforces-client/build/interfaces/standings.interface';
+
+export const GetAllProblems = async (client: CodeforcesClient, contestId: string) => {
+    const standingsResponse = await client.contest.standings({ contestId: contestId });
+    const problems: Problem[] = [];
+    let standings!: Standings;
+    if (standingsResponse.status === 'OK') {
```

Comment:
We should explicitly fail and exit the program if we failed to fetch from CF API.
If this response failed for any reason, we may fail anyways in a later step but we won't know the root cause of the issue, and if the program ran successfully, it will produce incorrect results any way.

Suggestion:

```js
else {
	console.error(...);
	process.exit(1);
}
```